src/chainrules/chainrules.jl

```diff
@@ -0,0 +1,12 @@
+export rrule
+"""
+    ChainRulesCore.rrule(itp::AbstractInterpolation, x...)
+ChainRulesCore.jl `rrule` for integration with automatic differentiation libraries.
+"""
+function ChainRulesCore.rrule(itp::AbstractInterpolation, x...)
+    y = itp(x...)
+    function pullback(Δy)
+        (NO_FIELDS, Δy * Interpolations.gradient(itp, x...))
```

Comment:
One quick thought (sorry this should have come earlier) `NO_FIELDS` -> `DoesNotExist()`. @rick2047 would you be up for a tiny follow up?

Reply:
sure, I was bothered by this too. but this is already merged, can I still push to this branch?

Reply:
You can use the same branch. It's your branch. Just send a new PR when you are ready.